Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -307,4 +307,66 @@ function CloningPod_JobComplete(j)
 	
 end
 
+function LandingPad_Temp_UpdateAction(furniture, deltaTime)
+	spawnSpot = furniture.GetSpawnSpotTile()	
+	jobSpot = furniture.GetJobSpotTile()
+	inputSpot = World.current.GetTileAt(jobSpot.X, jobSpot.y-1)
+    
+	if(inputSpot.inventory == nil) then
+		if(furniture.JobCount() == 0) then
+			itemsDesired = {Inventory.__new("Steel Plate", furniture.GetParameter("tradeinamount"), 0)}
+			
+			j = Job.__new(
+			inputSpot,
+			nil,
+			nil,
+			0.4,
+			itemsDesired,
+			Job.JobPriority.Medium,
+			false
+			)
+			
+			j.RegisterJobCompletedCallback("LandingPad_Temp_JobComplete")
+			
+			furniture.AddJob(j)
+		end
+	else
+		furniture.ChangeParameter("smelttime", deltaTime)
+		
+		if(furniture.GetParameter("smelttime") >= furniture.GetParameter("smelttime_required")) then
+			furniture.SetParameter("smelttime", 0)
+			
+		 --[[	outputSpot = World.current.GetTileAt(spawnSpot.X+2, spawnSpot.y)
```

Comment:
no :( I still get the issue, even while commenting more lua code

ScriptRuntimeException: attempt to index a nil value
MoonSharp.Interpreter.Execution.VM.Processor.ExecIndex (MoonSharp.Interpreter.Execution.VM.Instruction i, Int32 instructionPtr)
MoonSharp.Interpreter.Execution.VM.Processor.Processing_Loop (Int32 instructionPtr)


I'll add a check in the parameter get/change in c# to throw a proper log in that case, but I may have to pause a few hours first